rds-mysql-kfh-transform/main.go

```diff
@@ -0,0 +1,123 @@
+package main
+
+import (
+	"bytes"
+	"compress/gzip"
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"strings"
+
+	"github.com/aws/aws-lambda-go/events"
+	"github.com/aws/aws-lambda-go/lambda"
+	"github.com/honeycombio/agentless-integrations-for-aws/common"
+	"github.com/honeycombio/honeytail/event"
+	"github.com/honeycombio/honeytail/parsers/mysql"
+)
+
+type CloudWatchLogBody struct {
```

Comment:
Given we're already pulling in `aws-lambda-go/evens` here, I think you can use the `CloudwatchLogs*` structs from there: https://pkg.go.dev/github.com/aws/aws-lambda-go@v1.34.1/events#CloudwatchLogsData